packages/base/build.gradle.kts

```diff
@@ -11,194 +11,71 @@
-  iosArm64()
-  iosX64()
-  watchosArm32()
-  watchosArm64()
-  watchosX64()
-  tvosArm64()
-  tvosX64()
 
-  if (buildWasm) {
-    wasmJs {
-      nodejs()
-      d8()
-      browser()
-    }
-    wasmWasi {
-      // nothing at this time
-    }
-  }
+  jvmTest {
+    implementation(kotlin("stdlib-jdk8"))
+    implementation(kotlin("test-junit5"))
+    implementation(libs.junit.jupiter)
 
-  if (buildMingw) mingwX64()
+    runtimeOnly(libs.junit.jupiter.engine)
+    runtimeOnly(libs.logback)
+  }
 
-  sourceSets {
-    val commonMain by getting {
-      dependencies {
-        implementation(kotlin("stdlib"))
-        api(projects.packages.core)
-        implementation(libs.elide.uuid)
-        api(libs.kotlinx.collections.immutable)
-        api(libs.kotlinx.datetime)
-        implementation(libs.kotlinx.coroutines.core)
-        implementation(libs.kotlinx.serialization.core)
-      }
-    }
-    val commonTest by getting {
-      dependencies {
-        implementation(kotlin("test"))
-        implementation(kotlin("stdlib"))
-      }
-    }
-    val jvmMain by getting {
-      dependencies {
-        implementation(kotlin("stdlib-jdk8"))
-        api(libs.slf4j)
-        api(libs.jakarta.inject)
-        api(mn.micronaut.context)
-        api(libs.kotlinx.collections.immutable)
-        api(libs.kotlinx.datetime)
-        implementation(libs.kotlinx.serialization.core)
-        implementation(libs.kotlinx.coroutines.core.jvm)
-        implementation(libs.kotlinx.coroutines.jdk9)
-        implementation(libs.kotlinx.coroutines.slf4j)
-      }
-    }
-    val jvmTest by getting {
-      dependencies {
-        implementation(kotlin("stdlib-jdk8"))
-        implementation(kotlin("test-junit5"))
-        implementation(libs.junit.jupiter)
-        runtimeOnly(libs.junit.jupiter.engine)
-        runtimeOnly(libs.logback)
-      }
-    }
-    val jsMain by getting {
-      dependencies {
-        // KT-57235: fix for atomicfu-runtime error
-        api("org.jetbrains.kotlin:kotlinx-atomicfu-runtime:1.8.20-RC")
-        implementation(kotlin("stdlib-js"))
-        implementation(libs.kotlinx.coroutines.core.js)
-        implementation(libs.kotlinx.serialization.json.js)
-        api(libs.kotlinx.collections.immutable)
-        api(libs.kotlinx.datetime)
-      }
-    }
-    val jsTest by getting {
-      dependencies {
-        implementation(kotlin("stdlib-js"))
-        implementation(kotlin("test"))
-      }
-    }
-    val nativeMain by getting {
-      dependencies {
-        implementation(kotlin("stdlib"))
-        implementation(libs.kotlinx.coroutines.core)
-        api(libs.kotlinx.collections.immutable)
-        api(libs.kotlinx.datetime)
-      }
-    }
-    val nativeTest by getting {
-      dependencies {
-        implementation(kotlin("stdlib"))
-        implementation(kotlin("test"))
-      }
-    }
+  js {
+    // KT-57235: fix for atomicfu-runtime error
+    api("org.jetbrains.kotlin:kotlinx-atomicfu-runtime:1.8.20-RC")
```

Comment:
wonder if we can drop this now that we're on `1.9.x`

Reply:
I'll investigate this